### DISCUSSION TITLE
Development Setup?

### DISCUSSION THREAD

**Discussion by NoahMLoomis** [onesine/react-tailwindcss-datepicker#28] [General]
@onesine Can you share your development setup so we can have an easier time at developping features? It's slow going right now because when I test a change, I need to publish an npm package then download it in my project every time I make a change.

Is there a better way to do this? Maybe an addition to the README is needed? Or it can be covered in a contributing.md?

**Comment by onesine** [comment]
Hi @JefteCaro 
So sorry, I have been away for a while. I was very busy. I'm not available like I was when I spent almost all my time improving the package. I will be available very soon.

I have the same problem as you. To test a change, I release the package myself and get it back. We need to find a way to have a development server to test the package when we make a change. A development server similar to the one in create-react-app.

**Comment by JefteCaro** [comment]
Hi @onesine 
No worries, I know we're both busy with our day jobs 😆 

@NoahMLoomis 
Try this from the latest `master` branch
```sh
# Using npm
npm install && npm dev

# Using yarn
yarn install && yarn dev
```
and open your browser to `http://localhost:8888`

This will use the `next` from `devDependencies`

**Reply by onesine** [reply]
Very interesting @JefteCaro , this was not available before.

**Reply by NoahMLoomis** [reply]
@JefteCaro there seems to be an issue with tailwind not working ... are you able to run it correctly?
![image](https://user-images.githubusercontent.com/56049624/211026339-44ed4898-0844-40ce-9009-079fb8145255.png)

**Reply by JefteCaro** [reply]
Try the latest changes, just pushed fix

**Reply by NoahMLoomis** [reply]
Good now, thanks a bunch. I still think it would be useful to have a contributing.md at some point, or a dev setup in the readme. Just my opinion though 🤷

**Reply by JefteCaro** [reply]
I added `PlayGround` section in the `README.md`.

But, we'll have `Contributing.md` in the future, once I've initialized our automated e2e testing with `cypress`.
